packages/server/src/ChunkExtractor.js

```diff
@@ -205,7 +205,7 @@ class ChunkExtractor {
     inputFileSystem = fs,
```

Comment:
🤔 🤔 🤔 this symbol is __undefined__

Reply:
Undefined? fs is imported. Errors?

Reply:
🤦‍♂️ imported above the fold of github review